calico-enterprise/visibility/elastic/l7/configure.mdx

```diff
@@ -30,28 +30,25 @@ L7 logs are visible in the Manager UI, service graph, in the HTTP tab.
 
 ## Before you begin
 
-**Not supported**
-- GKE
+### Limitations
 
-**Limitations**
+* L7 logging is not compatible with a service mesh such as Istio.
+* L7 log collection is not supported for host-networked client pods.
+* When selecting and deselecting traffic for L7 log collection, active connections may be disrupted.
```

Comment:
This is true for all 3 L7 features too. Although pods will be recycled and rolled out after being selected for L7, there can be existing HTTP connections that get dropped and reconnected. So we haven't changed that, it was true in old model and new model. But it is true for all L7 docs and not just this one. (I wish we had a single doc :) )